homeassistant/components/remote/strings.json

```diff
@@ -38,11 +38,11 @@
     },
     "toggle": {
       "name": "[%key:common::action::toggle%]",
-      "description": "Toggles a device on/off."
+      "description": "Sends the turn on or turn off command to toggle a device."
```

Comment:
> Sends the turn on or turn off command

That seems incorrect. It sends a toggle, some devices even support that naturally. Besides, the description you have inserted now makes it unclear now.

Reply:
> In theory we don't send the toggle command, we send the on or off command based on the state

I have to correct myself, integrations are able to implement a toggle functionality, but in the majority of cases, this is using the default

Reply:
@joostlek  Thanks for clarifying, then I'll revert as requested.